bridgebot/bidding.py

```diff
@@ -27,6 +24,8 @@ def __init__(self, dealer):
             Players.WEST: []
         }
 
+    def __ns_first_bid(self, strain):
```

Comment:
If this is going to have members called `ns_first_bid` and `ew_first_bid`, maybe it should be called something else.